### PR TITLE
Release-Cut-20.2.0  Revised

### PR DESCRIPTION
**Pull request by hemantt**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/projects/RDM/versions/36004

### Change description ###
Revised Release 20.2.0, with changes outlined below 

Bug

    [RDM-8698] - Elasticsearch exact match queries on phone numbers with spaces never return results
    [RDM-9824] - ChallengeQuestion tab issues

Feature

    [RDM-7484] - Add Support for Concurrent Execution of Test Suites

Task

    [RDM-9200] - FTA for Swagger Documentation and API Specs - Data Store API
    [RDM-9533] - Improve performance of LS Sweeper
    [RDM-9694] - CCD Logstash to index cases in state DRAFT in Elasticsearch
    [RDM-9696] - Create Liquibase Changeset For Additional Indexes
    [RDM-9741] - Revert Changes of RDM-9338
    [RDM-9910] - Fix Logstash "ghost cases" indexing issue
    [RDM-10321] - Enable Persistent Queue for Logstash
    [RDM-10389] - Release 20.2 Perftest/Demo deployment and Testing

Improvement

    [RDM-9343] - Elasticsearch sorts are case-sensitive

TechDebt

    [RDM-9939] - Move Organisation Policy validation from DefaultValidateCaseFieldsOperation to a dedicated validator
    [RDM-9954] - Remove no longer needed "defaultValue" field from API response

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: elastic-search-support/src/main/resources/casesIndexSettings.json [hunk 1]
```diff
@@ -5,16 +5,37 @@
...
+          "lowercase",
+          "asciifolding"
+        ]
+      }
+  },
+  "mapping" : {
+    "total_fields" : {
+      "limit" : "8000"
     }
   }
```
Comment (danlysiak) [inline elastic-search-support/src/main/resources/casesIndexSettings.json:40]:
```suggestion
  }
```

File: repository/src/main/resources/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -96,7 +96,5 @@
     <include file="db/changelog/db.changelog-rdm-9453.xml"/>
     <include file="db/changelog/db.changelog-rdm-9031.xml"/>
     <include file="db/changelog/db.changelog-rdm-9555.xml"/>
-    <include file="db/changelog/db.changelog-rdm-9994.xml"/>
-    <include file="db/changelog/db.changelog-rdm-9893.xml"/>
-    <include file="db/changelog/db.changelog-rdm-10238.xml"/>
```
Comment (danlysiak) [inline repository/src/main/resources/db/changelog/db.changelog-master.xml:101]: Will need to add these deleted ones back in since already in master

**Review by danlysiak — CHANGES_REQUESTED** [review]

File: repository/src/main/resources/db/changelog/db.changelog-master.xml (hunk 2 above)
Comment (hemantt) [reply repository/src/main/resources/db/changelog/db.changelog-master.xml:101]: https://github.com/hmcts/ccd-definition-store-api/pull/834

**Review by danlysiak — APPROVED** [review]

**Review by danlysiak — APPROVED** [review]

**Review by mario-paniccia — APPROVED** [review]
👍

**Pull request merged**